specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-04-01/managedClusters.json

```diff
@@ -1478,6 +1478,10 @@
           "type": "string",
           "description": "Version of orchestrator specified when creating the managed cluster."
         },
+        "nodeImageVersion": {
```

Comment:
is this a readonly property? adding new property is suggesting to be done in new API version.

Reply:
https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines

Reply:
this is not readonly. User can set it to "latest". This is a setting to be added to 2020-04-01, which should be a new API version right?